authorizer-app-backend/src/main/java/org/radarbase/authorizer/Config.kt

```diff
@@ -32,13 +35,39 @@ data class Config(
 
 data class AuthorizerServiceConfig(
     var baseUri: URI = URI.create("http://0.0.0.0:8080/rest-sources/backend/"),
-    var advertisedBaseUri: URI? = null,
+    val advertisedBaseUri: URI? = null,
+    val frontendBaseUri: URI? = null,
```

Comment:
what is the difference between the two? one is backend and the other is frontend? (both are advertised ones I assume?)

Reply:
Indeed, the first one is the advertisedBaseUri for the current service; the other is where the frontend can be contacted by clients.